packages/components/notification/src/notify.ts

```diff
@@ -29,7 +29,7 @@ const notifications: Record<
 const GAP_SIZE = 16
 let seed = 1
 
-const notify: NotifyPartial = function (options = {}) {
+const notify: NotifyFn & Partial<Notify> = function (options = {}) {
```

Comment:
**Question: ** Why `Partial`?

Reply:
TypeScript can not detect typed methods is defined, so make it optional.